modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -0,0 +1,122 @@
+[[aura-api-authentication]]
+= Authentication
+:description: This page describes how to authenticate requests to the Aura API.
+
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+
+[TIP]
+====
+The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+====
+
+The Aura API uses OAuth 2.0 for API authentication.
+
+== Creating credentials
+
+. Navigate to the https://console.neo4j.io/[Neo4j Aura Console Account Details page] in your browser.
+. Select the *Create* button in the *OAuth 2.0 Clients* section.
+. Enter a *Client name*, and select *Create*.
+. Securely save the *Client ID* and *Client Secret* you are given in the resulting modal; you will need these for the next step.
+
+[CAUTION]
+====
+You cannot retrieve your secret after you close the modal, so save it securely.
+====
+
+== Authenticating API requests
+
+To authenticate API requests to the link:{neo4j-docs-base-uri}/aura/api/specification/[Aura API], you must provide a Bearer Token in the `Authorization` header of your request as shown below:
+
+`Authorization: Bearer <access_token>`
+
+[TIP]
+====
+You can use the *Authorize* button on the link:{neo4j-docs-base-uri}/aura/api/specification/[Aura API] page to authenticate and test endpoints directly using your client ID and client secret.
+====
+
+=== Token endpoint
+
+You can use the following `/oauth/token` endpoint to obtain a Bearer Token for authenticating API requests.
+
+Authentication to the token endpoint uses HTTP Basic Auth using your client ID as the username and client secret as the password.
+footnote:[If you wish to manually log in...]
```

Comment:
I feel like this is a bit too long for a footnote, but if we want to explain this in detail I don't think we can make it any shorter.

```suggestion
footnote:[The HTTP Basic authentication process uses the login credentials to generate an `Authorization` header. If you need to generate the header manually, you need to encode the `username:password` combination in Base64. For example, if your username and password are `myuser` and `mypassword` respectively, on a Linux or a macOS command line you can run the `echo -n myuser:mypassword | base64` command; the resulting string `bXl1c2VyOm15cGFzc3dvcmQ=` can be used in the header as `Authorization: Basic bXl1c2VyOm15cGFzc3dvcmQ=`].
```